js/index.js

```diff
@@ -1,42 +1,49 @@
 // ITERATION 1
-
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  let price = product.querySelector(".price span");
+  let quantity = product.querySelector(".quantity");
+  let priceVal = price.innerHTML;
+  let quantityVal = product.querySelector("input").value;
+  let subtotalVal = priceVal * quantityVal;
+  let subtotal = product.querySelector(".subtotal span");
```

Comment:
Could the name of this variable be more descriptive/corresponding to what it describes? For example "subtotalSpanElement".